software/sentryshot.yml

```
@@ -0,0 +1,12 @@
+name: "SentryShot"
+website_url: "https://github.com/SentryShot/sentryshot"
+source_code_url: "https://codeberg.org/SentryShot/sentryshot"
```

In my opinion if the github is just a mirror, the codeberg should always be used instead.

```suggestion
website_url: "https://codeberg.org/SentryShot/sentryshot"
source_code_url: "https://codeberg.org/SentryShot/sentryshot"
```